docs/source/pages/guides/realsense_comparison.rst

```diff
@@ -0,0 +1,492 @@
+     - `OAK-D-Lite <https://www.indiegogo.com/projects/opencv-ai-kit-lite/>`__
+     - `OAK-ToF <https://shop.luxonis.com/collections/beta-store/products/oak-ffc-tof>`__
+     - `L515 <https://www.intelrealsense.com/lidar-camera-l515/>`__
+     - `D415 <https://www.intelrealsense.com/depth-camera-d415/>`__
+     - `D430-D435 <https://www.intel.com/content/www/us/en/products/sku/98320/intel-realsense-depth-module-d430/specifications.html>`__
+     - `D450-D455 <https://www.intelrealsense.com/depth-camera-d455/>`__
+     - `F455 <https://ark.intel.com/content/www/us/en/ark/products/212561/intel-realsense-id-solution-f455.html>`__
+     - `F450 <https://www.intel.com/content/www/us/en/products/sku/212558/intel-realsense-id-solution-f450/specifications.html>`__
+     - `T261-T265 <https://www.intelrealsense.com/tracking-camera-t265/>`__
+   * - Custom AI models
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+   * - Object detection
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+   * - Object tracking
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+   * - On-device scripting
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+   * - Video/Image Encoding
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+   * - Image Manipulation
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+   * - Skeleton/Hand Tracking
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+   * - 3D Semantic segmentation
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+   * - 3D Object Pose Estimation
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+   * - 3D Edge Detection
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+   * - Feature Tracking
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ✔️
+   * - 3D Feature Tracking
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+   * - OCR
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+   * - Face Recognition
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ✔️
+     - ✔️
+     - ❌
+   * - Fiducial Marker localization
+     - ✔️
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+     - ❌
+   * - Active Stereo
```

Comment:
This being ❌ for Lite and OAK-D might lead to thinking no stereo. We should add Passive a passive block ? Or 
Have a depth Block and use some other representation inside the table for Active, Passive or stereo and have a asterisk below explaining what they means. Like they do in spec description in Phones or camera websites ?

Reply:
Maybe it would be better to just have `Depth Type` as in `Camera specification`, but I am not sure if we want to duplicate it. Should we just remove the Active Stereo / Laser depth rows?